geoips/schema/product_templates/alg_cmap.yaml

```diff
@@ -0,0 +1,36 @@
+type: object
+required:
+  - interface
+  - metadata
+  - spec
+properties:
+  interface:
+    type: string
+  metadata:
+    type: object
+    properties:
+      name:
+        type: string
+      family:
+        type: string
+      short-description:
+        type: string
+      long-description:
+        type: string
+  spec:
+    type: object
+    additionalProperties: false
```

Comment:
Note most of these have "additionalProperties: false".  The sectored*, unsectored*, xarray*, and unmodified* families have "additionalProperties: true"